src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,11 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> getAllMovie = session.createQuery("FROM Movie", Movie.class);
```

Comment:
Let's keep our SQL in one style - lower case for HQL/JPQL and upper case for pure SQL